yg-pe.py

```diff
@@ -43,14 +42,15 @@ def create_my_orders(self):
             # minimum size bumped if necessary such that you always profit
             # least 50% of the miner fee
             self.minsize = max(
-                int(1.5 * self.txfee / float(self.cjfee_r)), self.minsize)
+                int(1.5 * self.txfee / float(self.cjfee_r)), max_minsize)
```

Comment:
are you sure this is supposed to be max_minsize? Isn't self.minsize already set to that? I think this was right as it was